config/resource-manager.hcl

```diff
@@ -40,6 +40,11 @@ resource_manager "network" "2022-07-01" {
   readme_file_path = "../../swagger/specification/network/resource-manager/readme.md"
 }
 
+resource_manager "network" "2022-11-01" {
+  swagger_tag = "package-2022-11"
+  readme_file_path = "../../swagger/specification/network/resource-manager/readme.md"
+}
```

Comment:
`Network` is now available in `hashicorp/go-azure-sdk` and that should be used for new resources going forwards: https://github.com/hashicorp/go-azure-sdk/tree/main/resource-manager/network/2022-09-01

What's new/needed in this API Version out of interest?

Reply:
this Swagger fix only patched new API version https://github.com/Azure/azure-rest-api-specs/commit/d5606f4a04c76ad20ad0ca04a4c5c12bd2af3016

I noticed this version is in "ignore" line of Pandora, so I opened this PR.. https://github.com/hashicorp/pandora/blob/72e9024a7cdf44eab2ba50dccf547dbb52648bb1/config/resource-manager.hcl#L341

Reply:
Pretty sure that's only in the ignore list since we're not using that version right now, works fine for me locally:

> 2023-05-17T03:31:39.840+0200 [INFO]  Importer for Service "Network".Importer for API Version "2022-11-01": ✅ Service "Network" - Api Version "2022-11-01"

For now we'll import this both into Kermit and `go-azure-sdk` - but we'll be tackling moving `network` over to `go-azure-sdk` at some point in the not-too-distant future fwiw